Description: Don't fail build intentionally on unknown systems
Author: Balint Reczey <balint@balintreczey.hu>

Index: gpac/include/gpac/configuration.h
===================================================================
--- gpac.orig/include/gpac/configuration.h
+++ gpac/include/gpac/configuration.h
@@ -203,8 +203,8 @@ This section documents the base data typ
 #define GPAC_HAS_JPEG
 #define GPAC_HAS_PNG
 
-#else
-#error "Unknown target platform used with static configuration file"
+//#else
+//#error "Unknown target platform used with static configuration file"
 #endif
 
 
